Python/Movie-Info-Telegram-Bot/README.md

```diff
@@ -0,0 +1,44 @@
+# Movie Info Telegram Bot
+
+## Description
+A telegram Bot made using python which scrapes IMDb website and has the following functionalities
+1. Replies to a movie name with genre and rating of the movie
+2. Replies to a genre with a list of top movies and tv shows belonging to that genre
+
+## Setup Instructions
+1. Create a bot in telegram:
+
+       1. Go to @BotFather and click /start and type /newbot and give it a name. 
+       2. Choose a username and get the token 
+       3. Place the token in "YOUR-TOKEN" place in the code
+
+2. Install required packages:
+
+       pip install requests
+       pip install python-telegram-bot
+3. Run the python code to start the bot
+
+4. Type /start command to start conversation with the chatbot.
+
+5. Type /name <movie_name> to get the genre and Rating of the movie. The bot replies with atmost three results.
+6. Type /genre \<genre> to get a list of movies and TV shows belonging to that genres
+
+## Output
+
+### /start command
+
+![alt text](start.png)  
```

Comment:
We no longer keep images in the code base, kindly go through the CONTRIBUTION GUIDELINES before contributing.

Upload images on Imgur or postimages, paste their link in the README, and remove all the images from the codebase